### ISSUE TITLE
Doesn't seem to respect config.toml in .config

### ISSUE THREAD

**Issue by inglor** [orhun/rustypaste-cli#10]
The tool doesn't seem to respect the configuration in `$HOME/.config/rustypaste/config.toml`

**Comment by orhun** [comment]
Just tested this and couldn't reproduce.

https://github.com/orhun/rustypaste-cli/blob/267a96457413dbf2d1d03a1f5a13c1da8ed62693/src/lib.rs#L24

https://github.com/orhun/rustypaste-cli/blob/267a96457413dbf2d1d03a1f5a13c1da8ed62693/src/lib.rs#L34

Can you check again?

**Comment by orhun** [comment]
Updated README.md in 6ce0302e42a1d69d9b0eb91cfb8cd2605e94fa05

**Comment by angelacastaneda** [comment]
Any chance we could get $XDG_CONFIG_HOME to override macOS default when explicitly defined?

**Comment by orhun** [comment]
@angelacastaneda I'd prefer not to break the OS standards... you can report it to [upstream](https://github.com/xdg-rs/dirs) though.

**Comment by angelacastaneda** [comment]
macOS xdg support has been an issue for the last [five years](https://github.com/xdg-rs/dirs/issues/45). upstream seems [dead](https://github.com/xdg-rs/dirs/issues/60) :(

**Comment by orhun** [comment]
Ah damn... then I'd accept a PR for 1. switching to a maintained alternative of `dirs` 2. supporting XDG in MacOS (if the new upstream does not already support it)

**Comment by tessus** [comment]
I think it is a good compromise. Personally I hate `Library/Application Support` for terminal apps. The space alone drives me nuts (although newer macOS have a symlink without the space). (I created a symbolic link in `.config` that points to `Library/Application Support` and an alias to cd into `Library/Application Support`. It's not the best solution but it's a decent enough workaround.)

I have found that many terminal apps on macOS use in fact the `.config` dir (with a subdirectory) and don't even touch the `Library/Application Support` directory at all. Additionally they allow to override the config dir with `XDG_CONFIG_HOME`. e.g. some people use another config dir, like `.local/...` or `Library/Application Support` ;-)
